Sunrise/de.lproj/Localizable.strings

```diff
@@ -175,4 +175,16 @@
 /* Get Directions */
 "Get Directions" = "Anfahrt";
 
+/* Discounts */
+"Discounts" = "Rabatt";
```

Comment:
This should be the title of the dialogue showing textual representation of all the discounts which have been applied. So IMHO I'd use plural (Discounts or Coupons).